### PR TITLE
Allow install/launch on iOS using ios-deploy

### LINKED ISSUE #5392: Automatically install Android and iOS builds on devices
**Is your feature request related to a problem? Please describe (REQUIRED):**
Other engines provide an option to automatically install the built APK or APP files on Android or iOS devices.
Such option would speed up development and make it easier overall.

**Describe the solution you'd like (REQUIRED):**
Add a checkbox such as "Install and run on device" when building for iOS or Android. A default connected device is fine, but a dropdown with the list of devices is welcome too.

**Describe alternatives you've considered (REQUIRED):**
There is no alternative.

**Additional context (OPTIONAL):**
Unity does it for Android. Solar2D does it for both Android and iOS.

### PR DESCRIPTION
**Pull request by vlaaad**
Technical notes: I moved some stuff from `editor.adb` ns to `editor.fs` ns (`existing-path` check and `evaluate-path` helper).

User-facing changes: this changeset enhances the editor with functionality to install and launch bundled iOS applications on a connected iOS device. You need [ios-deploy](https://github.com/ios-control/ios-deploy) installed for this feature to work.

Fixes #5392

### CODE REVIEW

**Review by matgis — CHANGES_REQUESTED** [review]
Do we want to perhaps rename some things before merging?

File: editor/src/clj/editor/bundle_dialog.clj [hunk 1]
```diff
@@ -626,26 +643,61 @@
     (ui/value! provisioning-profile-text-field (or (get-string-pref prefs "bundle-ios-provisioning-profile")
                                                    (get-string-pref prefs "last-provisioning-profile")))
     (ui/value! architecture-64bit-check-box (prefs/get-prefs prefs "bundle-ios-architecture-64bit?" true))
-    (ui/value! architecture-simulator-check-box (prefs/get-prefs prefs "bundle-ios-architecture-simulator?" false))))
+    (ui/value! architecture-simulator-check-box (prefs/get-prefs prefs "bundle-ios-architecture-simulator?" false))
+    (ui/value! ios-deploy-install-check-box (prefs/get-prefs prefs "bundle-ios-ios-deploy-install" false))
+    (ui/value! ios-deploy-launch-check-box (prefs/get-prefs prefs "bundle-ios-ios-deploy-launch" false))))
```
[the remark this answers is not in the document]
Comment (vlaaad) [reply editor/src/clj/editor/bundle_dialog.clj:648]: Everything except `?` as a suffix for booleans makes sense. As [I've said before](https://vlaaad.github.io/2019-03-30/question-marks-in-clojure), `?` should be used as a suffix only for predicate functions, not for keywords that identify booleans. Maybe it's time to strip the `?` from the other `?`-suffixed settings?

[the remark this answers is not in the document]
Comment (matgis) [reply editor/src/clj/editor/bundle_dialog.clj:648]: I agree with your take on reserving `?` for predicate functions, but here we're talking about string keys in the preference store, so I don't think it necessarily applies here.

**Review by matgis — APPROVED** [review]
LGTM!

**Pull request merged**